js/libs.js

```diff
@@ -287,29 +287,31 @@ Fliplet.Registry.set('comflipletanalytics-report:1.0:core', function(element, da
         var momentTime;
 
         switch (dateSelectMode) {
-          case 'last-24-hours':
-            momentTime = moment(this.x).subtract(1, 'days');
-            break;
-          case 'last-7-days':
-            momentTime = moment(this.x).subtract(7, 'days');
-            break;
-          case 'last-30-days':
-            momentTime = moment(this.x).subtract(30, 'days');
-            break;
-          case 'last-90-days':
-            momentTime = moment(this.x).subtract(90, 'days');
-            break;
-          case 'last-6-months':
-            momentTime = moment(this.x).subtract(6, 'months');
-            break;
-          case 'last-12-months':
-            momentTime = moment(this.x).subtract(12, 'months');
-            break;
-          case 'custom-dates':
-            momentTime = moment(this.x).subtract(timeDeltaInMillisecs);
-            break;
+        case 'last-24-hours':
```

Comment:
is this in our rules? I believe `case` should be indented, please fix it as required

Reply:
@squallstar 
Yes, it an EsLint indent rule which is suggesting I fix it like this.

Code before the fix:
![image](https://user-images.githubusercontent.com/53430352/73748870-fde17600-4762-11ea-8bae-3175fa46398a.png)

Should we ignore errors like this?

Reply:
@YaroslavOvdii I have corrected the `indent` rule to allow 2 spaces for the case, see https://github.com/Fliplet/eslint-config-client/commit/171629ef4e210cd11f03db2c5b857dde8e675c2c

Update the `Fliplet/eslint-config-client` repo via npm (version 0.7) to get the update.

cc @tonytlwu @hcarneiro @MaksymShokin

Reply:
@squallstar 
Updated lint config and applied new indent rules.